app/javascript/components/TrackList.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Track } from './TrackList/Track'
+import { TextFilter } from './TextFilter'
+import { useList } from '../hooks/use_list'
+import { useRequestQuery } from '../hooks/request_query'
+
+export function TrackList(props) {
+  const [request, setFilter, setSort, setPage] = useList(props.request)
+  const { status, data, isFetching } = useRequestQuery('track-list', request)
+
+  return (
+    <div className="track-list">
+      <TextFilter
+        filter={request.query.filter}
+        setFilter={setFilter}
+        id="track-list-filter"
```

Comment:
See [this comment](https://github.com/exercism/v3-website/pull/93/files#r493478915) about preferring class names over IDs.